dashboard/app/serializers/workshop_teacher_serializer.rb

```diff
@@ -0,0 +1,7 @@
+class WorkshopTeacherSerializer < ActiveModel::Serializer
```

Comment:
I think we don't need the WorkshopTeacherSerializer anymore and can use the normal UserSerializer